tests/broker_test.go

```diff
@@ -53,12 +59,65 @@ func TestBrokerLogic(t *testing.T){
 					"credentials":"not-real-token",
 				},
 		}, 
-		// empty because we don't use it
 		&broker.RequestContext{})
 
 	// we want an error here
 	if errProvisionImproper == nil {
-		t.Errorf("Error on Provision with invalid token: no error returned")
+		t.Errorf("Error on Provision with invalid token: no error returned\n")
+	}
+
+	// Instance doesn't exist
+	_, errProvisionFake := businessLogic.Provision(
+		&osb.ProvisionRequest{
+			InstanceID:	"not-an-instance",
```

Comment:
Formatting is a mess here.  Has this run through go format?  Ideally a format check should be part of travis.